app/serializers/post_serializer.rb

```diff
@@ -549,6 +550,14 @@ def include_reviewable_score_pending_count?
     can_review_topic?
   end
 
+  def suspended
```

Comment:
```suggestion
  def user_suspended
```

I would follow the existing convention and prefix user attributes with `user_`. I feel like the meaning of the attribute is also clearer.